technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,29 @@
+# Performance Testing of TEAMMATES
+
+This report gives a detailed explanation of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes our proposed solution.
+
+* [Introduction](#Introduction)
```

Comment:
I think the previous paragraph already serves as an intro :P

We can flush it out more if we need to

Reply:
Yeah, noticed this after pushing. :P